website/documentation/050-tutorials/content/howto/shell-to-node/_index.md

```diff
@@ -0,0 +1,333 @@
+You can use the ops-toolbelt image that contains already a bunch of useful tools, or pick any other image.
+Specify the node from the drop-down menu that should troubleshooted.
+
+
+![76f62f8c5c8f6031fab21463ed1d35d6.png](_resources/f7b10d48edf44c17ba838ff5c429e39d.png)
+
+The dashboard then schedules a pod with the specifiied configurations and opens a shell session to the node.
+
+To get access to common binaries installed on the host - prefix command with "chroot /hostroot" (the path depends on where the root pasth is mounted in the container. For the ops-toolbelt image it is under /hostroot) 
+
+![0df1cc43eeafd164bfed1e47ea333487.png](_resources/3da659e9cc4744a2ad3e1c6a50d39c04.png)
+
+
+# Node fails to join the Cluster
+
+Kubernetes means cannot be used to troubleshoot a node that fails to join the cluster.
+Additionally, Gardener typically provisions the CloudProvider instances to be in a private subnet of the VPC - hence have no public IP address that could be used for direct SSH access.
+
+For this scenario, CloudProvider typically have extensive documentation (e.g [AWS](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstances.html) & [GCP](https://cloud.google.com/compute/docs/instances/connecting-to-instance) and in [some cases tooling support](https://cloud.google.com/compute/docs/instances/connecting-advanced#vpn)). 
+However, these approaches are mostly CloudProvider specific and require interaction via the CloudProvider CLI or API. 
+
+Instead you can use `gardenctl` that provides out-of-the-box support to ssh into an instance in a private subnet for all major Cloud Providers.
+
+## Identifying the problematic instance
+Firstly we need to pinpoint which instance is having problems. In Gardener, worker pools can be created in different CloudProvider regions, zones and accounts. So identifying the problematic instance via the CloudProvider is the first stepping-stone. 
+Even though having identified the correct CLoudProvider region and account of the Shoot Cluster, the instance would typically show up as sucessfully started / running in the CloudProvider dashboard / API - so it is not immediately obvious which instance has a problem.
+
+Instead we can use the Gardener API / CRDs to obtain the instance identifier in a Cloud-agnostic way.
+Gardener uses the [Machine Controller Manager](https://github.com/gardener/machine-controller-manager) to create the Shoot worker nodes. 
+For each worker node, the Machine Controller Manager creates a `Machine` CRD in the Shoot namespace in the respective Seed cluster. 
+Usually the problematic instance can be identified as the respective `Machine` CRD is `pending`.
+The `Machine` already contains the instance / node name in the .status, obtain it via:
+`k get machine <machine-name> -o json | jq -r .status.node`
+
+This is all the information needed to go ahead and use `gardenctl ssh` to get a shell to the node (you can skip to the section [Gardenctl ssh](#gardenctl-ssh) below). 
```

Comment:
The additional info below sounds interesting . why prompt to leave?
What may be necessary is to explain why would one need to do that?
For example, "To get additional details about the instance from the cloud provider, ..." or something like this.